collector/ethtool_linux.go

```diff
@@ -40,37 +40,48 @@ var (
 	transmittedRegex = regexp.MustCompile(`_tx_`)
 )
 
-type EthtoolStats interface {
+type Ethtool interface {
+	DriverInfo(string) (ethtool.DrvInfo, error)
 	Stats(string) (map[string]uint64, error)
 }
 
-type ethtoolStats struct {
+type ethtoolLibrary struct {
```

Comment:
Maybe call this just ethtool? But not feeling strongly

Reply:
I would like to call it that way, but it conflicts with the import:
```
collector/ethtool_linux.go:48:6: ethtool redeclared in this block
        previous declaration at collector/ethtool_linux.go:34:2
```